src/IconButton/IconButton.less

```diff
@@ -12,6 +12,8 @@
 	margin: 0 @moon-spotlight-outset;
 	line-height: @moon-icon-sprite-size;
 
+	.moon-taparea(@moon-icon-button-size);
```

Comment:
Is the idea here that if either `@moon-icon-button-size` or `@moon-button-small-tap-area-height` are altered such that a tap area would be required, it would automatically add it in?